app/templates/home-view.jade

```diff
@@ -1,331 +1,547 @@
-            div(data-i18n="new_home.courses_available_in")
-
-    .testimonials-rows
-      .testimonials-filler-left
-      .testimonials-filler-right
-      .row
-        .col-lg-offset-2.col-lg-7.col-sm-8
-          blockquote
-            h3(data-i18n="new_home.boast")
-
-        .col-lg-2.col-sm-3.text-center
-          img.img-circle(src="/images/pages/home/opensource.png")
-          h6 Open Source
-          .small opensource.com
-
-      .row
-        .col-lg-7.col-sm-8.col-sm-push-4.col-lg-push-3
-          blockquote
-            h3(data-i18n="new_home.winning")
-
-        .col-lg-2.col-sm-3.col-lg-offset-1.text-center.col-sm-pull-8.col-lg-pull-7
-          img.img-circle(src="/images/pages/home/pcmag.png")
-          h6 PC Mag
-          .small pcmag.com
-
-    .request-demo-row.text-center
-      h3(data-i18n="new_home.run_class")
-      if view.isTeacherWithDemo
-        div
-          button.btn.btn-primary.btn-lg.setup-class-btn(data-event-action="Homepage Setup Class Page Bottom", data-i18n="new_home.setup_a_class")
-      else
-        div
-          button.btn.btn-primary.btn-lg.request-demo(data-event-action="Homepage Request Demo Page Bottom", data-i18n="new_home.request_demo")
+          .text-center
+            button.btn.btn-lg.btn-primary.request-demo(data-i18n="new_home.request_quote")
+
+    .row.row-dark.text-center#built-for-teachers
+      .col-lg-12
+        h1.text-gold(data-i18n="new_home.built_for_teachers_title")
+        img(src="/images/pages/home/pink_rectangle.png")
+        .container
+          .row
+            .col-lg-12
+                p(data-i18n="new_home.built_for_teachers_blurb")
+                #built-for-teachers-carousel.carousel.text-left(data-interval=3000)
+                  ol.carousel-indicators
+                    li(data-target="#built-for-teachers-carousel", data-slide-to="0").active
+                    li(data-target="#built-for-teachers-carousel", data-slide-to="1")
+                    li(data-target="#built-for-teachers-carousel", data-slide-to="2")
+                  .carousel-inner
+                    .item.active
+                      .row.hidden-md.hidden-sm.hidden-xs
+                        .col-lg-6
+                          img.computer(src="/images/pages/home/built_for_teachers1.png")
+                        .col-lg-4
+                          h3.text-teal(data-i18n="new_home.built_for_teachers_subtitle1")
+                          p
+                            .text-navy(data-i18n="[html]new_home.built_for_teachers_subblurb1")
+                          if me.isAnonymous()
+                            br
+                            button.btn.btn-lg.btn-primary-alt.teacher-btn(data-event-action="Homepage Click Teacher Button CTA", data-i18n="[html]new_home.try_it_for_free")
+                        .col-lg-2
+                          p
+                            img(src="/images/pages/home/javascript_logo.png", height="60")
+                          p
```

Comment:
Is there a purpose for wrapping `img` with a `p` tag?
Is it just for layout purposes?

Reply:
Just layout, feel free to remove.